consensus/consensus_v2.go

```diff
@@ -211,6 +212,9 @@ func (consensus *Consensus) finalCommit() {
 		Int("numTxns", len(block.Transactions())).
 		Int("numStakingTxns", len(block.StakingTransactions())).
 		Msg("HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!")
+	ConsensusCounterVec.With(prometheus.Labels{"consensus": "hooray"}).Inc()
+	ConsensusGaugeVec.With(prometheus.Labels{"consensus": "block_num"}).Set(float64(block.NumberU64()))
+	ConsensusGaugeVec.With(prometheus.Labels{"consensus": "num_commits"}).Set(float64(consensus.Decider.SignersCount(quorum.Commit)))
```

Comment:
here the signers are already reset (after commitBlock). Better use the same signerCount at the beginning of finalCommit().

Reply:
fixed.